.github/workflows/test-build-deb.yml

```diff
@@ -218,14 +231,17 @@ jobs:
         java: [14]
     steps:
       - uses: actions/checkout@v1
+
       - uses: actions/checkout@v1
```

Comment:
Two checkout?

Reply:
Same thing 174-176

Reply:
Was already present! Didn't touch the existing code. though one checkouts the workflow repo and other the plugin repo. Not sure why we need both.

Reply:
Didnt realize it checkout 2 repos. This one resolved.